learn_ai/notebooks/chapter-05-Intro-to-Prompting.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Ae. This is adding a FileNotFoundError — please be more careful!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/FuturePathAI/Learn-AI-Engineering/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Yes , I Missed Pushing the fix with the error  😓. I've pushed now. </p>